plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -2084,6 +2093,21 @@ def updateBrushSize(self, value):
             effect = self._scribblesEditorWidget.activeEffect()
             effect.setParameter("BrushAbsoluteDiameter", value)
 
+    def createScribblesROINode(self):
+        scribblesROINode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsROINode")
+        scribblesROINode.AddObserver(scribblesROINode.PointPositionDefinedEvent, self.onROIPointPlaced)
```

Comment:
You are adding observers everytime this function is called, however I do not see a RemoveObserver logic anywhere? Maybe a good idea to cover that along with a `resetROI()` function....

Reply:
New logic looks good to me..